jfr-connection/build.gradle.kts

```diff
@@ -0,0 +1,12 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.publish-conventions")
+}
+
+description = "OpenTelemetry JFR Connection"
+otelJava.moduleName.set("io.opentelemetry.contrib.jfr.connection")
+
+dependencies {
+  testImplementation("org.openjdk.jmc:common:8.3.0")
+  testImplementation("org.openjdk.jmc:flightrecorder:8.3.0")
```

Comment:
@jack-berg It's to analyze the JFR events contained into a jfr file in tests

Reply:
@jack-berg In jfr streaming I think that the JFR events the events are not of the same type

Reply:
> @jack-berg In jfr streaming I think that the JFR events the events are not of the same type

@jack-berg Not the same Java objects

Reply:
Ok, so if it doesn't related to `jfr-streaming`, how do you anticipate it being used with OpenTelemetry?

Reply:
In the future, the JFR events provided from `jfr-connection` could be analyzed to produce telemetry data, such as `jfr-streaming` is doing. 

`jfr-streaming` uses JFR  event stream ([RecordingStream](https://cr.openjdk.org/~egahlin/jep-349/javadocs/api/jdk.jfr/jdk/jfr/consumer/RecordingStream.html)). `jfr-connection` allows having JFR events as an `InputStream` or a file. `jfr-connection` works for JVM >= 8 and `jfr-streaming` from JVM >= Java 17. The two projects seem complementary.

Reply:
> In the future, the JFR events provided from jfr-connection could be analyzed to produce telemetry data, such as jfr-streaming is doing.

Yes, and I think the analysis could use the same code. If you remotely connect to a process and save JFR files using `jfr-connection`, you can then open those files using [RecordingFile](https://docs.oracle.com/en/java/javase/11/docs/api/jdk.jfr/jdk/jfr/consumer/RecordingFile.html), and access the individual events as follows:
```
 try (RecordingFile recordingFile = new RecordingFile(Paths.get("recording.jfr"))) {
   while (recordingFile.hasMoreEvents()) {
     RecordedEvent event = recordingFile.readEvent();
     System.out.println(event);
   }
 }
```
If `jfr-streaming` was adjusted to allow the source of events to be configurable, then the same code used for `RecordingStream` to could be used to process events from `jfr-connection`. 

One caveat (there are likely others) is that `jfr-streaming` primarily records metrics. The metrics APIs do not have the ability to specify the time at which the measurement occurred - they're assumed to have happened whenever the API was invoked. This means that the time fields of the produced metrics would reflect when they were processed, or observed, rather than when the JFR events originally occurred.

Reply:
> how do you anticipate it being used with OpenTelemetry?

I would also expect this to be used with Otel-based continuous profiling, especially for Java 8.